### DISCUSSION TITLE
Default Labels feature for personal accounts

### DISCUSSION THREAD

**Discussion by lawgimenez** [community/community#45519] [Other Feature Feedback, Questions, & Ideas]
It would be nice if we could set default labels like in [organizations](https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization). So that every new repository I don't have to add another 15-20 new labels each time.

If this would be a premium feature, I won't mind.

**Comment by jge162** [comment]
it is possible yes. There are third-party tools like GitHub Label Sync that can be used to automatically add labels to new repositories. These tools work by syncing labels across multiple repositories. browse the GitHub marketplace.

 [Label Sync](https://github.com/marketplace/actions/label-sync) just search labels in marketplace.
